solidity/contracts/bank/Bank.sol

```diff
@@ -222,22 +222,23 @@ contract Bank is Ownable {
         );
         require(v == 27 || v == 28, "Invalid signature 'v' value");
 
-        bytes32 digest = keccak256(
```

Comment:
These changes look weird because it's exactly what was altered by linter in @nkuba' PR #158. I don't say we should block this PR but we need to figure out what is happening so that we do not start linting back and forth between PRs.